js/import/import.ui.js

```diff
@@ -591,7 +644,14 @@ const detectSections = async (src, frame) => {
       section.mapping = 'unset';
 
       if (!mappingData.find((m) => m.id === section.id)) {
-        mappingData.push(section);
+        mappingData.push({
+          id: section.id,
+          domId: section.domId,
+          domClasses: section.domClasses,
+          xpath: section.xpath,
+          layout: section.layout,
+          color: section.color,
+        });
```

Comment:
This is the selection from the 'box' to maintain and save to JSON.  I kept the `id` and `classes` separated for now - felt more future-proof - but might want to collapse at some point.